### ISSUE TITLE
Invalid access token

### ISSUE THREAD

**Issue by RacerXXX** [eracknaphobia/plugin.video.mlbtv#74]
Since 7/19/2024 (the return from the mlb all star break).  I get invalid access token after I pick the feed (home, away) 

I tried uninstalling the kodi plugin and then reinstalled it.  All games from all teams present and past are effected by the bug

Right now the only way I can watch is on my tablet thru mlb app from google play.

Thanks for creating the plugin for kodi it's great......

**Comment by Boschman** [comment]
I'm experiencing the same problem. It would be great if this could be fixed; I really love this plugin!

**Comment by eracknaphobia** [comment]
I'm working on a fix. Same issue reported in #73.

**Comment by eracknaphobia** [comment]
Please try the latest version (2024.7.22+matrix.1) it should fix this issue.

**Comment by RacerXXX** [comment]
I encountered this issue on my Xiaomi Mi Box S
Build:21.0 (21.0.1)
Git 2024409-7e760d1063
Compilied 2024-04-09

Kodi just ran an update the other day (first run),  I have the latest pluguin

**Comment by RacerXXX** [comment]
Tested on Windows 10 64bit

Kodi latest build Omega

same issue access token

**Comment by eracknaphobia** [comment]
Not the latest kodi build the latest version of this addon. You have to download it from the github page as a zip and install manually.

https://github.com/eracknaphobia/plugin.video.mlbtv/archive/refs/heads/master.zip

**Comment by RacerXXX** [comment]
I tested out Windows 10

KodiSetup-20240718-edf6da24-master-x64

I install the master.zip file mlbtv plugin

I does not error out with invalid access token, however when I go thru the menu and select 1 inning it hour glasses and then hangs kodi so it still seems to be off a bit.....

**Comment by RacerXXX** [comment]
I did a test on my Xiaomi Mi Box S

I unistalled the mlbtv pluging and installed the master zip mlbtv plugin

It allows me to select everything up the inning,  When I select the 1st inning it paused for  quite a while and then the game with the time comes up. It show negative time for the beginning and the correct end time for the end of the game,  Wait a little longer and the time corrects to the beginning zero of the game.  It's like I  never picked the inning.  But the game does play.  So the menu for inning is being ignored,  Also it's pretty latent waiting for the game to start.
